postgres-exporter/include/entrypoint

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+function append_ssl_mode {
```

Comment:
what do we need this for? If I understand how this works correctly it just uses what is in the ENV var for `DATA_SOURCE_NAME` shouldn't that have that correct value ssl or not?

Reply:
because the connection string does not always include the SSL option. for example in dev cloud the secret does not include the ?sslmode=disable argument on the connection string, but still the DB does not have SSL enabled

Reply:
We discussed in Zoom just now and decided let's add a comment here on the difference between DATA_SOURCE_NAME vs DATA_SOURCE_URI.